vendors/circom/circomlib/circuit/quadratic_arithmetic_program.h

```diff
@@ -28,9 +28,11 @@ class QuadraticArithmeticProgram {
     using Evals = typename Domain::Evals;
     using DensePoly = typename Domain::DensePoly;
 
-    std::pmr::vector<F> a(domain->size());
-    std::pmr::vector<F> b(domain->size());
-    std::pmr::vector<F> c(domain->size());
+    std::vector<Evals> abc;
+    abc.reserve(3);
```

Comment:
```suggestion
    std::vector<Evals> abc(3);

Reply:
This will construct evals as if I did `abc.resize(3)`. Behavior is different from `reserve()`. And following `emplace_back`s would make the size 6.